Tests/LibGLSL/test-parser.cpp

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2021, Itamar S. <itamar8910@gmail.com>
```

Comment:
Is your name Itamar? :^)

Reply:
The code has a lot of copy-pasted parts (or even full files) from LibCpp, which was created by Itamar, so I decided to leave his name there. Similarly, Andreas' and several other developers' names are there too.